### PR TITLE
answer to Q2

### PR DESCRIPTION
**Pull request by iryboto**
answered second question

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,9 @@ If you have any other questions not listed here, please raise a PR to have it ad
 
 - Q: I made a typo in my commit message, how can I __amend__ it?
 
+
```
Comment (iryboto) [inline README.md]: Please, remove this line.

File: README.md [hunk 2]
```diff
@@ -20,6 +22,7 @@ If you have any other questions not listed here, please raise a PR to have it ad
 - Q: How can I see a __diff__ of what's changed in the files I've changed but not committed? How about just the files I've staged?
 
 - Q: Can Git __show__ me what's changed in the previous commit?
+- A: type in git diff
```
Comment (iryboto) [inline README.md]: This is not correct. Please remove.

File: README.md [hunk 3]
```diff
@@ -20,6 +21,7 @@ If you have any other questions not listed here, please raise a PR to have it ad
 - Q: How can I see a __diff__ of what's changed in the files I've changed but not committed? How about just the files I've staged?
 
 - Q: Can Git __show__ me what's changed in the previous commit?
+- A: Type in git show
```
Comment (xavez88) [inline README.md]: Please can you format `git show` too ?

**Review by cranbim — APPROVED** [review]
Just testing my ability to review.
Looks OK to me

**Review by xavez88 — APPROVED** [review]

**Pull request merged**